source/use_case_ndvi_investigate.rst

```diff
@@ -0,0 +1,146 @@
+Investigation of vegetation health variations in a given area based on measurements from Sentinel-2
+"""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""
+
+
+Use Case Goal
+=============
+
+.. Required
```

Comment:
Remove the template text?

Reply:
It is just a comment which will not be seen on readhtedocs, right? Then, I think it can stay there for reference in case we want to change the use case description at some point..